packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -75,7 +75,7 @@ function createReconnectError(prefix: string, err: any) {
     const error = CreateContainerError(err);
     const error2 = Object.create(error);
     error2.message = `${prefix}: ${error.message}`;
-    error2.canRetry = true;
+    error2.canRetry = error2.canRetry ?? true;
```

Comment:
I do not think this line is needed here. Removing it completely will have same impact.

Reply:
Removed the change.